doc-content/enterprise-only/admin-and-config/proc-configuring-kie-server-persistence.adoc

```diff
@@ -0,0 +1,66 @@
+[id='proc-configuring-kie-server-persistence_{context}']
+
+= Configuring {KIE_SERVER} Persistence
+
+You can configure the {KIE_SERVER} persistence by passing hibernate or JPA parameters as system properties.
+
+To pass the hibernate or JPA parameters, use the following prefixes in your system properties:
+
+* `javax.persistence`
+* `hibernate`
+
+.Procedure
+
+. To configure {KIE_SERVER} persistence, complete any of the following tasks:
+
++
+--
+If you want to configure {KIE_SERVER} persistence using {EAP} configuration file, complete the following tasks:
+
+... In your {PRODUCT} installation directory, navigate to the `standalone-full.xml` file. For example, if you use {EAP} installation for {PRODUCT}, go to `$EAP_HOME/standalone/configuration/standalone-full.xml` file.
+... Open the `standalone-full.xml` file and under the `<system-properties>` tag, set your hibernate or JPA parameters as system properties.
++
+The following example shows how to configure {KIE_SERVER} persistence using hibernate parameters:
++
+[source,xml,subs="attributes+"]
+----
+<system-properties>
+    ...
+        <property name="hibernate.hbm2ddl.auto" value="create-drop"/>
+    ...
+<system-properties>
+----
++
+The following example shows how to configure {KIE_SERVER} persistence using JPA parameters:
++
+[source,xml,subs="attributes+"]
+----
+<system-properties>
+    ...
+        <property name="javax.persistence.jdbc.url" value="jdbc:mysql://mysql.db.server:3306/my_database?useSSL=false&serverTimezone=UTC"/>
+    ...
+<system-properties>
+----
+
+
+If you want to configure {KIE_SERVER} persistence using command line, complete the following tasks:
+
+... Pass the parameters directly from the command line using `-Dkey=value` as follows:
```

Comment:
If there is only one step, then I believe numbering is not required. @emmurphy please confirm.